superset/reports/commands/alert.py

```diff
@@ -84,12 +85,12 @@ def run(self) -> bool:
         except (KeyError, json.JSONDecodeError) as ex:
             raise AlertValidatorConfigError() from ex
 
-    def _validate_not_null(self, rows: np.recarray) -> None:
+    def _validate_not_null(self, rows: NDArray[Any]) -> None:
```

Comment:
This could have been typed as `np.recarray[Any, Any]`, but would have required Python >= 3.9. So for now we need to fall back to `NDArray[Any]`.